pkg/daemon/ceph/osd/encryption.go

```diff
@@ -32,6 +38,39 @@ func closeEncryptedDevice(context *clusterd.Context, dmName string) error {
 		return errors.Wrapf(err, "failed to close encrypted device. %s", cryptsetupOut)
 	}
 
-	logger.Info(cryptsetupOut)
+	logger.Infof("dm version:\n%s", cryptsetupOut)
+	return nil
+}
+
+func dmsetupVersion(context *clusterd.Context) error {
```

Comment:
Why do we need to print the dmsetup version?

Reply:
It's useful for debugging and cheap to run.